security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,12 +15,21 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-4"]
+== OpenShift File Integrity Operator 1.3.4
+
+The following advisory is available for the OpenShift File Integrity Operator 1.3.4:
+
+* link:https://access.redhat.com/errata/RHBA-2023:5652[RHBA-2023:5652 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+
+This update addresses a bug in an underlying dependency.
+
 [id="file-integrity-operator-release-notes-1-3-3"]
 == OpenShift File Integrity Operator 1.3.3
 
 The following advisory is available for the OpenShift File Integrity Operator 1.3.3:
 
-* link:https://access.redhat.com/errata/RHBA-2023:5652[RHBA-2023:5652 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
+* link:https://access.redhat.com/errata/RHBA-2024:2946[RHBA-2024:2946 OpenShift File Integrity Operator Bug Fix and Enhancement Update]
```

Comment:
This file is not found. You might want to check the link?

Reply:
@stevsmit that change is bogus. Thank you for catching it!